charts/sonarqube-dce/templates/config.yaml

```diff
@@ -9,18 +9,15 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 data:
-  {{- if and .Values.sonarSecretKey (not .Values.ApplicationNodes.sonarProperties) }}
-  sonar.properties: sonar.secretKeyPath={{ .Values.sonarqubeFolder }}/secret/sonar-secret.txt
-  {{- end }}
-  {{- if or .Values.ApplicationNodes.sonarProperties }}
+{{- if or .Values.sonarSecretKey .Values.ApplicationNodes.sonarProperties }}
   sonar.properties:
-  {{ range $key, $val := .Values.ApplicationNodes.sonarProperties }}
+  {{- range $key, $val := .Values.ApplicationNodes.sonarProperties }}
```

Comment:
Is it working fine if .Values.ApplicationNodes.sonarPropertie is not specified at all?

Reply:
Yes it output nothing if it is empty